apps/stats/src/app.jsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * Global polyfills
+ */
+import '@automattic/calypso-polyfills';
+
+import ReactDom from 'react-dom';
+import { QueryClient, QueryClientProvider } from 'react-query';
+import { Provider } from 'react-redux';
+import { createStore, applyMiddleware, compose } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+// import StatsSummary from 'calypso/my-sites/stats/index';
+import { rawCurrentUserFetch, filterUserObject } from 'calypso/lib/user/shared-utils';
+import StatsOverview from 'calypso/my-sites/stats/overview.jsx';
+import analyticsMiddleware from 'calypso/state/analytics/middleware';
+import consoleDispatcher from 'calypso/state/console-dispatch';
+import { setCurrentUser } from 'calypso/state/current-user/actions';
+import currentUser from 'calypso/state/current-user/reducer';
+// import ui from 'calypso/state/ui/reducer';
+import wpcomApiMiddleware from 'calypso/state/data-layer/wpcom-api-middleware';
+import { setStore } from 'calypso/state/redux-store';
+import sites from 'calypso/state/sites/reducer';
+// import stats from 'calypso/state/stats/reducer';
+import { combineReducers, addReducerEnhancer } from 'calypso/state/utils';
+import registerStatsPages from './routes';
+import 'calypso/assets/stylesheets/style.scss';
+
+async function AppBoot() {
+	const rootReducer = combineReducers( {
+		currentUser,
+		sites,
```

Comment:
You'll probably need to add the `calypso/state/stats` reducer to make the Stats UIs really work. Without the Redux state, they won't be able to fetch stats data.

Reply:
stats states are automatically registered when use individual states by the stats components. Will look at see what we could do to make it explicit.

Reply:
Of course, most reducers are dynamically registered 🤦‍♂️ I forgot about it despite working on the dynamic registration myself 🙂